engine-tests/src/tests/erc20_connector.rs

```diff
@@ -852,7 +857,12 @@ pub mod sim_tests {
             input: balance_tx.data,
         });
         let result = aurora.call("call", &call_args.try_to_vec().unwrap());
-        let submit_result: SubmitResult = result.unwrap_borsh();
+        let submit_result = match result.status() {
```

Comment:
This change (and the corresponding ones in `sanity.rs` and `xcc.rs`) is needed because `near-sdk-sim` is still on borsh v0.9. I figured changing this callsite was easier than going through the process of updating `near-sdk-sim` since it will be replaced with `aurora-workspace` soon.

Reply:
I did consider doing that. But it didn't seem necessary when all the `near-sdk-sim` code is going to be removed soon.